src/MoBi.Presentation/Presenter/EditExplicitFormulaPresenter.cs

```diff
@@ -201,7 +200,7 @@ public void Validate(string formulaString)
             _view.SetParserError(null);
             _context.PublishEvent(new FormulaValidEvent(_formula, BuildingBlock));
          }
-         catch (FuncParserException parserException)
+         catch (OSPSuiteException parserException)
```

Comment:
What is the reason for this change?

Reply:
FuncParserException  comes from FuncParser which cannot be referenced from the presenter layer (AnyCPU vs x64 bit).

Reply:
And the validation method now throws an OSPSuiteException so this would not even work anymore :)